input-gen-runtimes/rt-input-gen.cpp

```diff
@@ -375,6 +383,11 @@ struct InputGenRTTy {
     return V;
   }
 
+  template <> FunctionPtrTy *getNewValue<FunctionPtrTy *>(bool Stub, int Max) {
```

Comment:
just a nitpick, maybe make have the * in FunctionPtrTy like it is in VoidPtrTy just to have things consistent